docs/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 2.29.1
+
+- `Fix` — Toolbox wont be shown when Slash pressed with along with Shift or Alt
```

Comment:
Control or meta keys

Reply:
`CMD+/` and `CTRL+/` open Block Tunes — this behaviour stays the same.
Problem was in `Shift+/` and `Alt+/` — Toolbox won't be shown now, since "key" will be  "?" or "÷" — we don't need to check Shift or Alt separately. Previously we checked `keyCode` which is the same for those chars.

Reply:
Oh, okay. Shift is a bit weird for me here, because in some layouts I can't type / without shift.